### DISCUSSION TITLE
IsMantlingAllowedToStart() as public or protected function?

### DISCUSSION THREAD

**Discussion by colorindarkness** [Sixze/ALS-Refactored#106] [Q&A]
Would it be possible to make CanMantle() function so it can disable TryStartMantlingInAir() on Tick as well?  Currently it only effects Grounded Mantling and there is no way to override it from the Input. 
> if (CanMantle())
> {
> TryStartMantlingInAir();
> }

**Comment by Sixze** [comment]
Not really sure what you're talking about, there is no `CanMantle()` function, only `IsMantlingAllowedToStart()` which is called in any case.

**Reply by colorindarkness** [reply]
My bad! Instead could IsMantlingAllowedToStart() be made protected or public? I am developing a plugin that extends yours and I'm trying not to need to edit your code. 
https://github.com/Voidware-Prohibited/ALSXT

**Reply by Sixze** [reply]
Yes, of course, you can make the functions you need public or protected and send me a pull request with your changes.
